package.json

```diff
@@ -51,7 +51,7 @@
     "registry": "https://registry.npmjs.org/"
   },
   "dependencies": {
-    "@eppo/js-client-sdk-common": "4.12.0",
+    "@eppo/js-client-sdk-common": "4.13.2",
```

Comment:
https://github.com/Eppo-exp/js-sdk-common/releases/tag/v4.13.2

Release with the fix